src/build-time-render/BuildTimeRender.ts

```diff
@@ -19,6 +19,7 @@ import {
 
 import renderer, { Renderer } from './Renderer';
 import * as cssnano from 'cssnano';
+import * as ora from 'ora';
```

Comment:
do we think build time render should be doing the actual logging?

Reply:
It's only using the types from `ora`, as it's really just interacting with an API on the passed object. That being said, since that's already the case I could just remove the import and pass a callback or something if that'd be preferable.

Reply:
sorry late reply on this. I meant more as in we're referencing things such as `spinnerText`, which seems very logging specific to me? Perhaps we should come up with a real logging api now?

Reply:
@matt-gadd At what level do you mean? Just for our plugins, for cli-build-app, or all of Dojo cli? Happy to work on that but I think I'd need to better understand what you'd have in mind before I could come up with a useful solution.

Reply:
I added a very basic logging API for live updates. Provided by webpack-contrib and used in cli-build-app so webpack-contrib modules can use the type without a circular dependency.